templates/.circleci/config.yml.tpl

```diff
@@ -3,6 +3,7 @@
 version: 2.1
 {{- $prereleases := stencil.Arg "releaseOptions.enablePrereleases" }}
 {{- $testNodeClient := and (has "grpc" (stencil.Arg "serviceActivities")) (has "node" (stencil.Arg "grpcClients")) }}
+{{- $defaultBranch := .Git.DefaultBranch | default "main" }}
```

Comment:
not sure if it's possible to default `.Git.DefaultBranch` with `main` for unit test?

Reply:
I have a ticket somewhere to have a default value for that :/

Reply:
yeh I recalled that